api/v1alpha1/podtypes/access_policy.go

```diff
@@ -3,13 +3,14 @@ package podtypes
 // +kubebuilder:object:generate=true
 type AccessPolicy struct {
 	//+kubebuilder:validation:Optional
-	Inbound InboundPolicy `json:"inbound,omitempty"`
+	Inbound *InboundPolicy `json:"inbound,omitempty"`
 	//+kubebuilder:validation:Optional
 	Outbound OutboundPolicy `json:"outbound,omitempty"`
 }
 
 // +kubebuilder:object:generate=true
 type InboundPolicy struct {
+	//+kubebuilder:validation:Required
 	Rules []InternalRule `json:"rules"`
```

Comment:
Is adding required here a breaking change?

Reply:
My presumption during the last PR was that no one has InboundPolicy set without also setting InternalRules, which would not make it breaking. However, the non-pointer on the InboundPolicy made it so it is somehow set in spite of not defining it in the Application Spec.

Currently it is already required, setting this flag just made it explicit, as the default behaviour in the API is required values unless otherwise specified